R/label-utils.R

```diff
@@ -6,6 +6,40 @@
 #' @param data_to_label newly transformed dataset
 #' @param variable_details variable_details.csv
 #' @param variables_sheet variables.csv
+#' 
+#' @return labeled data_to_label
+#' 
+#' @examples 
+#' library(cchsflow)
+#' bmi2010 <- rec_with_table(
+#'   data = cchs2010, variable_details =
+#'     variable_details, dataset_name = "cchs2010", variables = c(
+#'     "HWTGHTM",
+#'     "HWTGWTK", "HWTGBMI_der"
+#'   )
+#' )
+#'
+#' head(bmi2010)
+#'
+#' bmi2012 <- rec_with_table(
+#'   data = cchs2012, variable_details =
+#'     variable_details, dataset_name = "cchs2012", variables = c(
+#'     "HWTGHTM",
+#'     "HWTGWTK", "HWTGBMI_der"
+#'   )
+#' )
+#'
+#' tail(bmi2012)
```

Comment:
Is it necessary to show head and tails of the transformed data before labelling? Maybe only showing a head of the combined data after applying labels to it.

Reply:
perhaps replace the head and tails with get_label() before merge after merge then after setting